bnd.bnd

```diff
@@ -0,0 +1,10 @@
+Bundle-SymbolicName: org.jspecify.jspecify
+Bundle-Name: JSpecify annotations
+Bundle-Vendor: jspecify.dev
```

Comment:
```suggestion
Bundle-Vendor: jspecify.org
```

Would it be nicer to use `org` consistently?

Reply:
> we should just use "The JSpecify Authors," as in our source files?

I will change it to "The JSpecify Authors"

Reply:
Changed it to "The JSpecify Authors" in 87ce0eb